### PR TITLE
Add alternative solutions

### PR DESCRIPTION
**Pull request by theindra**
Added a section "Alternative Solutions" to capture conversations at TPAC 2024

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,15 @@ With the above steps in place we pave the way to change the default to stop auto
 3. Deprecation period: gradually change behavior to default-off.
 
 
+# Alternatives Solutions
+As discussed during TPAC 2024, an alternative to the above described flow could be to make Autofill a two-stage process. The JS handler is called when the country has been filled, giving the website a chance to update it's UX. In the JS handler the website indicates when the website has been updated which gives the browser a signal when to continue autofilling.
```
Comment (yoavweiss) [inline README.md:71]: Maybe link to the minutes?

**Review by theindra — COMMENTED** [review]
Add TPAC slides and meeting minutes

File: README.md (hunk 1 above)
Comment (theindra) [reply README.md]: Good point. Added them

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**